### PR TITLE
Add html file, css file and js files

### PR DESCRIPTION
**Pull request by wolterbwambale**
This is my portfolio project where I have made the following:-
I have added the popup window,
responsiveness, mobile menu validation form, and desktop version and button hovering.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+              <li class="lang-list">JavaScript</li>
+              
+            </ul>
+            <a href="#">
+              <button class="see-btn">see live <img src="./img/social-media-icons/live.png" alt="live">
+              </button></a>
+            <a href="#"><button class="see-btn">see sorce <img src="./img/social-media-icons/github.png" alt="see more">
+            </button></a>
+        </span> 
+      </div>  -->
```
Comment (Nemwel-Boniface) [inline index.html]: - [ ] Kindly ensure that you remove all commented-out/ unused/ unwanted pieces of code from your code base. It makes your codebase look messy and cluttered. Having commented out code also violates the [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+    privately personalized reads; no accounts
+     or sign-ups required.</p>
+    <ul class="lang-tag">
+        <li class="lang-list">HTML</li>
+        <li class="lang-list">CSS</li>
+        <li class="lang-list">JavaScript</li>
+    </ul>
+<button class="see-btn">see project</button>
+</div>  
+<!-- </div> -->
```
Comment (Nemwel-Boniface) [inline index.html]: - [ ] Kindly ensure that you remove all commented-out/ unused/ unwanted pieces of code from your code base. It makes your codebase look messy and cluttered. Having commented out code also violates the [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: App.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  event.preventDefault();
+  if (message.value === '') {
+    errorMsg.textContent = 'Please enter a message';
+  } else if (email.value.toUpperCase() === email.value) {
+    errorMsg.textContent = 'please email must not be capitalized';
+  } else {
+    errorMsg.textContent = '';
+    form.submit();
+  }
+});
```
Comment (Nemwel-Boniface) [inline App.js]: - [ ] For some reason, your form still accepted uppercase email addresses to be sent. Kindly look into why this is happening and resolve it to meet the project requirements.

**For a visual, this is how I am seeing it:**

[Screencast from 06-04-2023 04:47:15 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/230397496-a6e707bf-c319-45a0-9d75-b328503332cb.webm)

Ideally the form should be blocked from submission. You could use the "[Prevent default](https://www.w3schools.com/jsref/event_preventdefault.asp)" function to implement this.

**Review by Nemwel-Boniface — COMMENTED** [review]
## Hello @wolterbwambale 

### This is a manual review

# :recycle: Request Changes :recycle: 

Cheers and Happy coding!

**Comment by wolterbwambale** [comment]
Hello @Nemwel-Boniface i have update the required changes

**Review by Nemwel-Boniface — COMMENTED** [review]
## Hello @wolterbwambale 

### This is a manual review

# :heavy_check_mark:  Approved :heavy_check_mark: 

Thank you for making the changes I requested :clap:  . I also hope that you were able to learn something new and apply it :slightly_smiling_face: 

Cheers and Happy coding!

**Pull request merged**